docs/user/dashboard/dashboard.asciidoc

```diff
@@ -116,56 +158,99 @@ To compare the data in the panels, arrange the panels on the dashboard, or remov
 
 * To resize, click the resize control, then drag to the new dimensions.
 
+* To maximize the panel to fullscreen, open the panel menu, then click *More > Maximize panel*.
+
 * To delete, open the panel menu, then select *Delete from dashboard*.
```

Comment:
This is actually nested under 'More' currently. Ideally, we will change that, but I don't see a fix for it getting in before 7.12.0